test/test_image.py

```diff
@@ -0,0 +1,473 @@
+import os
+from shlex import split
+from subprocess import PIPE, CalledProcessError, run
+
+import pytest
+
+from docker_cli.exceptions import CommandNotFoundOnImageError
+from docker_cli.image import Image, get_image_id
+
+
+@pytest.fixture
+def test_image_id():
+    """
+    Builds an image for testing and returns its ID.
+    """
+    run(split("docker build ./ -t test"))
+    inspect_process = run(
+        split("docker inspect -f='{{.Id}}' test"),
+        capture_output=True,
+        text=True,
+        check=True,
+    )
+    return inspect_process.stdout.strip()
+
+
+def test_init(test_image_id):
+    """
+    Tests that the __init__ function on the Image class is correctly
+    receiving and remembering the ID of a docker image.
+    """
+    id = test_image_id
+    print("ID: " + id)
+    img = Image("test")
+
+    assert img is not None
+    assert img.id == id
+
+    img_2 = Image(id)
+    assert img_2.id == id
+
+    run(split("docker image remove test"))
+
+
+def test_init_CalledProcessError():
```

Comment:
If possible, I'd like to conform to [PEP 8 naming conventions](https://peps.python.org/pep-0008/#naming-conventions) and avoid mixing underscores with camelcase.

```suggestion
def test_bad_init():
```